lib/pkg.ml

```diff
@@ -379,11 +399,24 @@ let prepare_opam_for_distrib ~version ~content =
   let without_version = List.filter is_not_version_field content in
   Fmt.str "version: \"%a\"" Version.pp version :: without_version
 
+let opam_file_contents name =
+  let filename = name |> Fpath.v |> Fpath.add_ext ".opam" in
+  let location =
+    OS.File.exists filename >>= function
```

Comment:
given you're already 4.08+ you can take the opportunity to use `let*`/`let+` instead of `>>=`/`>>|` for new code, but that's not mandatory either